packages/database/lib/migrations/20250227180909_sync_variant_unique.cjs

```diff
@@ -0,0 +1,20 @@
+exports.config = { transaction: false };
+
+const TABLE = '_nango_syncs';
+
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    await knex.schema.raw(`ALTER TABLE "${TABLE}" DROP CONSTRAINT IF EXISTS "_nango_syncs_name_nango_connection_id_deleted_at_unique";`);
+    await knex.schema.raw(`DROP INDEX CONCURRENTLY IF EXISTS "_nango_syncs_variant_name_nango_connection_id_deleted_at_unique"`);
```

Comment:
The original unique constraint and index I added when introducing the variant column are actually not working as expected. By default pg treats NULLs as distinct values. Since pg15 there is a `NULLS NOT DISTINCT` option that I could have use below but a partial index also does the job